lib/internal/Magento/Framework/App/Feed.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\App;
+
+/**
+ * Default XML feed class
+ */
+class Feed implements FeedInterface
+{
+    /**
+     * @param Zend_Feed $feed
+     * @param array $data
+     */
+    public function __construct(
+        \Zend_Feed $feed,
+        array $data
+    ) {
+       $this->feed = $feed;
+       $this->data = $data;
+    }
+
+    /**
+     * Returns the formatted feed content
+     * 
+     * @return string
+     */
+    public function getFormatedContentAs(
```

Comment:
Looks like there is a mix up between this method and the one in the interface is `getFormattedContentAs` but this one has only 1 t

Reply:
@dmanners yes, thanks :)